### PR TITLE
[RFC] Remove ':Print' command

### PR DESCRIPTION
**Pull request by ghost**
The documentation itself says it's deprecated, and I'm not convinced this isn't some long-running joke:

```
  "Just as ':print'.  Was apparently added to Vi for
  people that keep the shift key pressed too long..."
  Note: A user command can overrule this command.
```

I know there's many options which are kept for compatibility, but I just don't see much of one for this command.

### CODE REVIEW

File: runtime/doc/usr_40.txt [hunk 1]
```diff
@@ -290,8 +290,8 @@ deletes the first line.
 
 	Note:
 	User-defined commands must start with a capital letter.  You cannot
-	use ":X", ":Next" and ":Print".  The underscore cannot be used!  You
-	can use digits, but this is discouraged.
+	use ":Next".  The underscore cannot be used!  You can use digits, but
```
Comment (jusga) [inline runtime/doc/usr_40.txt]: You've removed `":X"`. Was that intentional?

Comment (Pyrohh) [inline runtime/doc/usr_40.txt]: Yes, the command itself was removed a while ago, but this piece of documentation must have been missed.

File: runtime/doc/map.txt [hunk 2]
```diff
@@ -1133,8 +1133,7 @@ All user defined commands must start with an uppercase letter, to avoid
 confusion with builtin commands.  Exceptions are these builtin commands:
 	:Next
 	:X
```
Comment (jusga) [inline runtime/doc/map.txt]: According to your comment below, I think this `":X"` can be removed, too.

Comment (Pyrohh) [inline runtime/doc/map.txt]: :+1:

**Comment by Pyrohh** [comment]
@fmoralesc Would you mind looking at beb793f09cae70f744228c16dbb3e302c1874aa3? I'm not sure if you recommend maybe making a `vimOnlyStdPlugin` group instead, or something...

**Comment by fmoralesc** [comment]
@Pyrohh since there's so few of these I would leave things as you have now, but `vimOnlyStdPlugin` could also work.

**Comment by Pyrohh** [comment]
Okay, thanks. I'll leave it as is for now.

**Comment by justinmk** [comment]
LGTM. After this, let's wait until after 0.1 before removing anything else.

**Comment by Pyrohh** [comment]
Thanks @justinmk, merged. I forgot to mention it in vim_diff.txt, so I just added that.

**Pull request closed without merging**